content/en/tracing/visualization/service.md

```diff
@@ -41,69 +44,135 @@ Enable them directly or create your own [APM monitors][3].
+* The times at which traces that correspond to this version were first and last seen.
+* An Error Types indicator, which shows how many types of errors appear in each version that did not appear in the immediately previous version.
 
-### Columns
+    **Note**: This indicator shows errors that were not seen in traces from the previous version. It doesn’t mean that this version necessarily introduced these errors. Looking into new error types can be a great way to begin investigating errors.
 
-Choose what to display in your resources list:
+* Requests per second.
+* Error rate as a percentage of total requests.
 
-* **Requests**: Absolute amount of requests traced (per seconds)
-* **Avg/p75/p90/p95/p99/Max Latency**: The Avg/p75/p90/p95/p99/Max latency of your traced requests
-* **Total time**: Sum of all time spend in this resource
-* **Error**: Absolute amount of error for a given resource
-* **Error Rate**: Percent of error for a given resource
+You can add columns to or remove columns from this overview table and your selections will be saved. The additional available columns are:
+
+* Endpoints that are active in a version that were not in the previous version.
+* Time active, showing the length of time from the first trace to the last trace sent to Datadog for that version.
+* Total number of Requests.
+* Total number of Errors.
+* Latency measured by p50, p75, p90, p95, p99, or max.
+
+{{< img src="tracing/visualization/service/deployments.png" alt="Deployments"  style="width:90%;">}}
+
+Read more about Deployments on the service page [here][12].
+
+### Error Tracking
+View issues on your service, which are similar errors aggregated together to turn a noisy stream of errors into manageable issues, and help you assess the impact of your service’s errors. Read more about issues in [Error Tracking][9])
+
+This tab has overview graphs that show which resources have the most issues, and a list of the most common issues occurring in your service. Click on an issue in the list to see details in a side panel, including its stack trace, related code versions, and total error occurrences since inception.
+
+{{< img src="tracing/visualization/service/error_tracking_side_panel.jpg" alt="Error Tracking tab"  style="width:90%;">}}
+
+### Infrastructure
+If your service is running on Kubernetes, you’ll see an Infrastructure tab on the Service Page. The live Kubernetes pods table shows you detailed information on your pods, such as if memory usage is close to its limit, and allows you to improve resource allocation by clearly seeing if provisioned compute resources exceed what’s required for optimal application performance. 
+
+{{< img src="tracing/visualization/service/infra_pods.png" alt="Kubernetes Pods"  style="width:90%;">}}
+
+The Kubernetes metrics below show you a high level summary of your infrastructure health for the selected time period, including CPU, Memory, Network, and Disk metrics.
```

Comment:
```suggestion
The Kubernetes metrics below show you a high-level summary of your infrastructure health for the selected period, including CPU, Memory, Network, and Disk metrics.
```

Reply:
I think we should leave "time period" in there for clarity